setup.cfg

```diff
@@ -0,0 +1,557 @@
+;        deprecated-itertools-function,
+;        deprecated-types-field,
+;        next-method-defined,
+;        dict-items-not-iterating,
+;        dict-keys-not-iterating,
+;        dict-values-not-iterating,
+;        deprecated-operator-function,
+;        deprecated-urllib-function,
+;        xreadlines-attribute,
+;        deprecated-sys-function,
+;        exception-escape,
+;        comprehension-escape
+
+# Enable the message, report, category or checker with the given id(s). You can
+# either give multiple identifier separated by comma (,) or put this option
+# multiple time (only on the command line, not in the configuration file where
+# it should appear only once). See also the "--disable" option for examples.
+enable=c-extension-no-member
+
+
+[REPORTS]
+
+# Python expression which should return a note less than 10 (10 is the highest
+# note). You have access to the variables errors warning, statement which
+# respectively contain the number of errors / warnings messages and the total
+# number of statements analyzed. This is used by the global evaluation report
+# (RP0004).
+evaluation=10.0 - ((float(5 * error + warning + refactor + convention) / statement) * 10)
+
+# Template used to display messages. This is a python new-style format string
+# used to format the message information. See doc for all details.
+#msg-template=
+
+# Set the output format. Available formats are text, parseable, colorized, json
+# and msvs (visual studio). You can also give a reporter class, e.g.
+# mypackage.mymodule.MyReporterClass.
+output-format=text
+
+# Tells whether to display a full report or only the messages.
+reports=no
+
+# Activate the evaluation score.
+score=yes
+
+
+[REFACTORING]
+
+# Maximum number of nested blocks for function / method body
+max-nested-blocks=5
+
+# Complete name of functions that never returns. When checking for
+# inconsistent-return-statements if a never returning function is called then
+# it will be considered as an explicit return statement and no message will be
+# printed.
+never-returning-functions=sys.exit
+
+
+[BASIC]
+
+# Naming style matching correct argument names.
+argument-naming-style=snake_case
+
+# Regular expression matching correct argument names. Overrides argument-
+# naming-style.
+#argument-rgx=
+
+# Naming style matching correct attribute names.
+attr-naming-style=snake_case
+
+# Regular expression matching correct attribute names. Overrides attr-naming-
+# style.
+#attr-rgx=
+
+# Bad variable names which should always be refused, separated by a comma.
+bad-names=foo,
+          bar,
+          baz,
+          toto,
+          tutu,
+          tata
+
+# Naming style matching correct class attribute names.
+class-attribute-naming-style=snake_case
+
+# Regular expression matching correct class attribute names. Overrides class-
+# attribute-naming-style.
+#class-attribute-rgx=
+
+# Naming style matching correct class names.
+class-naming-style=PascalCase
```

Comment:
In den Qualitätsrichtlinien steht CamelCase als Standard. Sehe aber gerade, dass das eh nicht üblich für Python ist. Sollte dann besser in den Qualitätsrichtlinien selbst als hier geändert werden.

Reply:
Ja hatte gedacht, dass das in den Qualitätsrichtlinien einfach ein Missverständnis war wie camelCase funktioniert. PascalCase ist ja genauso, nur, dass auch der erste Buchstabe groß geschrieben wird. Ist auch definitiv Konvention in Python